guides/tutorial/subscriptions.md

```diff
@@ -0,0 +1,144 @@
+# Subscriptions
+
+When the need arises for near realtime data GraphQL provides subscriptions. We want to support subscriptions that look like
+
+
+```graphql
+subscription{
+  newPost {
+    id
+    name
+  }
+}
+```
+
+Since we had already setup mutations to handle creation of posts we can use that as the event we want to subscribe to. In order to achieve this we have to do a little bit of set up
+
+
+Let's start by adding `absinthe_phoenix` as a dependency
+
+In `mix.exs`
+
+```elixir
+defp deps do
+  [
+    {:absinthe_phoenix, "~> 1.4.0"}
+    << other deps >>
+  ]
+```
+
+Then we need to add a supervisor to run some processes for the to handle result broadcasts
+
+In `lib/blog/application.ex`:
+
+```elixir
+  children = [
+    <<other supervisors>>
+
+    supervisor(Absinthe.Subscription, [BlogWeb.Endpoint])
```

Comment:
You may want to include the Elixir 1.5 style children here too.

Reply:
👍